cfgov/unprocessed/apps/analytics-gtm/webpack-config.js

```diff
@@ -35,9 +35,8 @@ const COMMON_MODULE_CONFIG = {
       loader: 'babel-loader?cacheDirectory=true',
       options: {
         presets: [ [ 'babel-preset-env', {
-          targets: {
-            browsers: BROWSER_LIST.LAST_2_IE_9_UP
-          },
+          configPath: __dirname,
+          useBuiltIns: 'usage',
```

Comment:
This uses `babel-polyfill` to inject polyfills where needed based on browser-support level. Should this be on for all our configs?